src/containers/AddItemScreen.tsx

```diff
@@ -0,0 +1,229 @@
+}
+
+const AddItem = ({
+    navigation,
+    botId
+} : AddItemProps) => {
+    const[nameErrorMessage, setNameErrorMessage] = useState("");
+    const[costErrorMessage, setCostErrorMessage] = useState("");
+    const[quantityErrorMessage, setQuantityErrorMessage] = useState("");
+
+
+    const[hasPermission, setHasPermission] = useState<boolean | null>(null);
+    const[type, setType] = useState(Camera.Constants.Type.back);
+    const[camera, setCamera] = useState<Camera|null>(null);
+    const[photo, setPhoto] = useState<string | null>(null);
+    const[itemName, setItemName] = useState("");
+    const[cost, setCost] = useState(0);
+    const[quantity, setQuantity] = useState(1);
+
+
+
+    const takePhoto = async () => {
+        if (camera) {
+            const pic = await camera.takePictureAsync({skipProcessing:true});
+            setPhoto(pic.uri);
+        }
+    };
+
+    const pickImage = async () => {
+        const pic = await ImagePicker.launchImageLibraryAsync({
+          mediaTypes: ImagePicker.MediaTypeOptions.Images,
+        });
+        if (pic.cancelled){
+            setPhoto(null);
+        } else {
+            setPhoto(pic.uri);
+        }
+    };
+    useEffect(() => {
+        (async () => {
+          const response: PermissionResponse = await Camera.requestPermissionsAsync();
+          setHasPermission(response.status === 'granted');
+        })();
+    }, []);
+
+    if (hasPermission === null) {
+        return (
+			<Text> Requesting permission </Text>
+        );      
+    }
+    if (hasPermission === false) {
+        return <Text> No access to camera </Text>;
+    }
+
+    const submitItem = async () => {
+        if (photo === null){
+            Alert.alert("Please submit a photo!")
+        } else if (itemName == "" || cost === 0 || quantity === 0) {
+            Alert.alert("Please fill out all inputs before submitting")
+        } else if (nameErrorMessage !== "" ||  quantityErrorMessage !== "" || costErrorMessage !== "") {
+            Alert.alert("Please fix errors before submitting")
+        } else {
+            //DO REQUEST
+            Alert.alert("Added Item succesfully");
+            navigation.navigate('InventoryModification');
+        }
+    }
+
+	return (
+    <>
+    <View style={{ flex: 1}}>
+    <FontAwesome name="arrow-left" style={
+                    { color: "black", 
+                    fontSize: 30,
+                    position: 'absolute',
+                    top: 40,
+                    left: 15,
+                }} onPress={()=>{
+                    navigation.navigate('InventoryModification')
+                }}/>
+        <Text style = {styles.header}> Add an Item </Text>
+        <View style = {styles.cameraContainer}>
+            { (photo === null) ? (
+            <>
+            <Camera style={styles.camera} type={type} 
+            ref={ref =>{
+                setCamera(ref);
+            }}/>
+            <Ionicons name="ios-photos" style={{ 
```

Comment:
I would recommend using icons from React Native Elements and choosing a consistent icon set (I think we're just sticking to FontAwesome for now)

https://reactnativeelements.com/docs/icon
Icon name: https://fontawesome.com/icons?d=gallery